contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue

```diff
@@ -642,10 +642,18 @@
         return (this.firstNode && this.getContentNodeFiles(this.firstNode.id)) || [];
       },
       fileDuration() {
+        let audioVideoFiles;
         if (this.firstNode.kind === 'audio' || this.firstNode.kind === 'video') {
```

Comment:
Again, you didn't modify this, but it's nice to [use constants](https://github.com/learningequality/studio/blob/unstable/contentcuration/contentcuration/frontend/shared/leUtils/ContentKinds.js#L18) for these so we can identify content kinds are utilized.